assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -165,15 +139,14 @@ function InnerElement({
         alt={alt}
         aria-label={alt}
         loading={'lazy'}
-        onClick={onClick(thumbnailURL, mediaBaseColor.current)}
         onLoad={makeMediaVisible}
-        {...dropTargetsBindings(thumbnailURL)}
+        draggable={false}
       />
     );
+    cloneProps.src = thumbnailURL;
   } else if (type === 'video') {
-    const { lengthFormatted, poster, mimeType } = resource;
     const displayPoster = poster ? poster : newVideoPosterRef.current;
-    return (
+    media = (
       <>
         <Video
```

Comment:
Found another weird thing for this element. 🤷 

When you abort dragging a video (by dropping the element while the mouse is still over the original element), the dragged clone static element seems to return to the original position on top of the video, but not with the exact same dimension - notice the right-most 1px of the image still animating and also the round corner disappearing (because the clone doesn't have that).

You have to move the mouse out of the element again to reset it.

![abort-video-bug](https://user-images.githubusercontent.com/637548/101862078-92fd8a00-3b3f-11eb-8a96-582b47b81e22.gif)